### ISSUE TITLE
TS Server fatal error:  Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by adambrennankainos** [microsoft/vscode#116425]
Issue Type: <b>Bug</b>

❗️❗️❗️ Please fill in the sections below to help us diagnose the issue ❗️❗️❗️

**TypeScript Version:** 4.1.4

**Steps to reproduce crash**
1. Start VSCode
2. Open a TS File
3. I receive the error stating that "The TypeScript language service died 5 times right after it got started. The service will not be restarted."

**TS Server Log**
❗️ Please review and upload this log file to help us diagnose this crash:
`c:\Users\xxx\AppData\Roaming\Code\logs\20210211T082845\exthost1\vscode.typescript-language-features\tsserver-log-oBPjhZ\tsserver.log`
The log file may contain personal data, including full paths and source code from your workspace. You can scrub the log file to remove paths or other personal information.

**TS Server Error Stack**
Server: `semantic`
```
TypeError: Cannot read property 'indexOf' of undefined
    at Object.parsePackageName (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:39472:30)
    at ConfiguredProject.Project.enablePlugin (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:151962:24)
    at ConfiguredProject.enablePluginsWithOptions (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:152477:30)
    at ProjectService.loadConfiguredProject (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:154206:25)
    at ProjectService.createAndLoadConfiguredProject (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:154145:22)
    at ProjectService.createLoadAndUpdateConfiguredProject (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:154150:36)
    at ProjectService.assignProjectToOpenedScriptInfo (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:155055:44)
    at c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:155395:97
    at Object.flatMap (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:513:25)
    at ProjectService.applyChangesInOpenFiles (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:155395:41)
    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:156413:46)
    at c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158286:88
    at IOSession.Session.executeWithRequestId (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158277:28)
    at IOSession.Session.executeCommand (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158286:33)
    at IOSession.Session.onMessage (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158310:35)
    at Interface.<anonymous> (c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:160520:27)
    at Interface.emit (events.js:315:20)
    at Interface._onLine (readline.js:329:10)
    at Interface._normalWrite (readline.js:474:12)
    at Socket.ondata (readline.js:186:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)
```

VS Code version: Code 1.53.1 (5d424b828ada08e1eb9f95d6cb41120234ef57c7, 2021-02-08T23:29:42.785Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.81GB (2.90GB free)|
|Process Argv|--crash-reporter-id baf21157-a02b-4d6a-b229-34708ad66af0|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.1
vscode-eslint|dba|2.1.14
prettier-vscode|esb|5.9.1
vscode-nunjucks|exe|0.1.0
vscode-jest-runner|fir|0.4.31
terraform|has|2.6.0
remote-wsl|ms-|0.53.4
vsliveshare|ms-|1.0.3629
vscode-jest|Ort|3.2.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30211402
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt600cf:30251589

```

</details>

**Comment by Mottoweb** [comment]
I am having the same issue, here is my log just in case.

`TypeError: Cannot read property 'indexOf' of undefined
        at Object.parsePackageName (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:39472:30)
        at ConfiguredProject.Project.enablePlugin (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:151962:24)
        at ConfiguredProject.enablePluginsWithOptions (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:152477:30)
        at ProjectService.loadConfiguredProject (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:154206:25)
        at ProjectService.createAndLoadConfiguredProject (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:154145:22)
        at ProjectService.createLoadAndUpdateConfiguredProject (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:154150:36)
        at ProjectService.assignProjectToOpenedScriptInfo (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:155055:44)
        at /Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:155395:97
        at Object.flatMap (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:513:25)
        at ProjectService.applyChangesInOpenFiles (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:155395:41)
        at Session.handlers.ts.Map.ts.getEntries._a.<computed> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:156413:46)
        at /Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158286:88
        at IOSession.Session.executeWithRequestId (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158277:28)
        at IOSession.Session.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158286:33)
        at IOSession.Session.onMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158310:35)
        at Interface.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:160520:27)
        at Interface.emit (events.js:315:20)
        at Interface._onLine (readline.js:329:10)
        at Interface._normalWrite (readline.js:474:12)
        at Socket.ondata (readline.js:186:10)
        at Socket.emit (events.js:315:20)
        at addChunk (_stream_readable.js:295:12)
        at readableAddChunk (_stream_readable.js:271:9)
        at Socket.Readable.push (_stream_readable.js:212:10)
        at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)
Info 29   [12:09:16.291] response:
    {"seq":0,"type":"response","command":"updateOpen","request_seq":2,"success":false,"message":"Error processing request. Cannot read property 'indexOf' of undefined\nTypeError: Cannot read property 'indexOf' of undefined\n    at Object.parsePackageName (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:39472:30)\n    at ConfiguredProject.Project.enablePlugin (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:151962:24)\n    at ConfiguredProject.enablePluginsWithOptions (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:152477:30)\n    at ProjectService.loadConfiguredProject (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:154206:25)\n    at ProjectService.createAndLoadConfiguredProject (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:154145:22)\n    at ProjectService.createLoadAndUpdateConfiguredProject (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:154150:36)\n    at ProjectService.assignProjectToOpenedScriptInfo (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:155055:44)\n    at /Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:155395:97\n    at Object.flatMap (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:513:25)\n    at ProjectService.applyChangesInOpenFiles (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:155395:41)\n    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:156413:46)\n    at /Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158286:88\n    at IOSession.Session.executeWithRequestId (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158277:28)\n    at IOSession.Session.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158286:33)\n    at IOSession.Session.onMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:158310:35)\n    at Interface.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:160520:27)\n    at Interface.emit (events.js:315:20)\n    at Interface._onLine (readline.js:329:10)\n    at Interface._normalWrite (readline.js:474:12)\n    at Socket.ondata (readline.js:186:10)\n    at Socket.emit (events.js:315:20)\n    at addChunk (_stream_readable.js:295:12)\n    at readableAddChunk (_stream_readable.js:271:9)\n    at Socket.Readable.push (_stream_readable.js:212:10)\n    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)"}
Info 30   [12:09:16.292] event:
    {"seq":0,"type":"event","event":"typingsInstallerPid","body":{"pid":29136}}
Info 31   [12:09:16.293] request:
    {"seq":3,"type":"request","command":"configurePlugin","arguments":{"pluginName":"@vsintellicode/typescript-intellicode-plugin","configuration":{"modelPath":"/Users/fedorbuturlin/.vscode/extensions/visualstudioexptteam.vscodeintellicode-1.2.11/cache/1A3F919CD5B7725108B03C349630A1C82B03_4A84987F2428442A9EC84563BB2BC063"}}}
Info 32   [12:09:16.304] response:
    {"seq":0,"type":"response","command":"configurePlugin","request_seq":3,"success":true}
Perf 33   [12:09:16.304] 3::configurePlugin: async elapsed time (in milliseconds) 11.3632
Info 34   [12:09:16.304] request:
    {"seq":4,"type":"request","command":"geterr","arguments":{"delay":0,"files":["/Users/fedorbuturlin/dev/audienzz/js/apps/ad-forms/src/state.tsx","/Users/fedorbuturlin/dev/audienzz/js/apps/ad-forms/src/global.d.ts","/Users/fedorbuturlin/dev/audienzz/js/apps/ad-forms/src/components/AdBanner/index.tsx"]}}
Perf 35   [12:09:16.305] 4::geterr: async elapsed time (in milliseconds) 0.6189
Info 36   [12:09:16.311] event:
    {"seq":0,"type":"event","event":"requestCompleted","body":{"request_seq":4}}`

**Comment by IllusionMH** [comment]
/duplicate of #116219 
Fix should go out with VS Code 1.53.2
